### PR TITLE
[16.0][MIG] - geoengine_partner

### PR DESCRIPTION
**Pull request by sbejaoui**

### CODE REVIEW

File: geoengine_partner/views/partner.xml
Comment (kouffsamuel) [inline geoengine_partner/views/partner.xml:30]: Maybe the geo_point field is missing here
`<field name="geo_point" />`

**Review by kouffsamuel — CHANGES_REQUESTED** [review]

File: geoengine_partner/views/partner.xml
Comment (sbejaoui) [reply geoengine_partner/views/partner.xml:30]: thanks,

**Review by lmignon — APPROVED** [review]
LGTM (code review + functional)

**Comment by ferdymercury** [comment]
Thanks for this work!
Is there something else missing here before merge? Can I help somehow?

**Comment by bealdav** [comment]
Hi probably a rebase could help here @sbejaoui Thanks

**Review by rousseldenis — APPROVED** [review]
Code review

File: geoengine_partner/views/partner.xml
Comment (kouffsamuel) [inline geoengine_partner/views/partner.xml]: Here, if we want to make the view editable, we need to add the attribute _editable_ for example : `<geoengine editable="1">`

File: geoengine_partner/views/partner.xml
Comment (kouffsamuel) [inline geoengine_partner/views/partner.xml:61]: We need to add `<field name="active_on_startup">True</field>` to show the layer when we go on the view.

File: geoengine_partner/views/partner.xml
Comment (kouffsamuel) [inline geoengine_partner/views/partner.xml:65]: We also need to add a starting opacity value for example : `<field name="layer_opacity">0.8</field>`

File: geoengine_partner/views/partner.xml
Comment (kouffsamuel) [inline geoengine_partner/views/partner.xml]: We can change the sequence to 1 as it will be the first layer to be displayed. ` <field eval="1" name="sequence" />`

**Review by kouffsamuel — CHANGES_REQUESTED** [review]

**Comment by sbejaoui** [comment]
@OCA/geospatial-maintainers , 
This PR is ready

**Review by kouffsamuel — APPROVED** [review]
Code review + functional

File: geoengine_partner/i18n/geoengine_partner.pot
Comment (IT-Ideas) [inline geoengine_partner/i18n/geoengine_partner.pot:7]:
```suggestion
"Project-Id-Version: Odoo Server 16.0\n"
```
?

File: geoengine_partner/README.rst
Comment (IT-Ideas) [inline geoengine_partner/README.rst:32]: Is this still valid?

**Review by IT-Ideas — COMMENTED** [review]
Thanks!

**Pull request merged**
